src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -231,6 +223,12 @@ class device {
         xclClose(m_handle);
     }
 
+    std::string name() const {
```

Comment:
Instead of reading it from sysfs every time, should we cache it since it won't change?

Reply:
Yes. Sounds good